### DISCUSSION TITLE
[Q/A][Java SDK (Component)] vitual threads support

### DISCUSSION THREAD

**Discussion by leyou240** [apache/dubbo#14729] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

Do you now support setting Dubbo's consumer and provider threads as custom virtual thread pools? Since our main services have been migrated to JDK 21.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment, marked as answer]
You can configure `dubbo.protocol.threadpool=virtual`
